app/services/encryption/multi_region_kms_client.rb

```diff
@@ -0,0 +1,78 @@
+require 'json'
+module Encryption
+  class MultiRegionKMSClient
+    def initialize
+      @aws_clients = {}
+      # Instantiate an array of aws clients based on the provided regions in the environment
+      JSON.parse(Figaro.env.aws_kms_regions).each do |region|
+        @aws_clients[region] = Aws::KMS::Client.new(
+          instance_profile_credentials_timeout: 1, # defaults to 1 second
+          instance_profile_credentials_retries: 5, # defaults to 0 retries
+          region: region, # The region in which the client is being instantiated
+        )
+      end
+    end
+
+    # Use each KMS client to encrypt with params and options
+    # Region format example: '{"reg": {"us-east-1": "cipher", "us-west-2": "othercipher"}}'
+    def encrypt(key_id, plaintext, encryption_context)
+      region_ciphers = {}
+      @aws_clients.each do |region, kms_client|
+        r_cipher = kms_client.encrypt(key_id: key_id,
+                                      plaintext: plaintext,
+                                      encryption_context: encryption_context)
+        region_ciphers[region] = r_cipher.ciphertext_blob
+      end
+      { reg: region_ciphers }.to_json
```

Comment:
is "reg" our key? or something from AWS? Could we spell out the full `region` to be clearer?

Reply:
reg is ours -- I'll go ahead and change that to region